hbase-thrift/src/main/resources/hbase-webapps/thrift/thrift.jsp

```diff
@@ -31,9 +31,12 @@ long startcode = conf.getLong("startcode", System.currentTimeMillis());
 String listenPort = conf.get("hbase.regionserver.thrift.port", "9090");
 String serverInfo = listenPort + "," + String.valueOf(startcode);
 ImplType implType = ImplType.getServerImpl(conf);
-String framed = implType.isAlwaysFramed()
-    ? "true" : conf.get("hbase.regionserver.thrift.framed", "false");
-String compact = conf.get("hbase.regionserver.thrift.compact", "false");
+String framed = (implType.isAlwaysFramed()
+    ? "true" : conf.get("hbase.regionserver.thrift.framed", "false"))
```

Comment:
"true" -> "Framed"?

nit: I think the logic has become nested and is unreadable for a ternary operator. Mind switching to an if-else?